kerastuner/tuners/bayesian.py

```diff
@@ -190,22 +199,30 @@ def _vectorize_trials(self):
                 else:
                     trial_value = hp.default
 
-                # Embed an HP value into a continuous space.
-                if isinstance(hp, hp_module.Boolean):
-                    val = 1 if trial_value else 0
-                elif isinstance(hp, hp_module.Choice):
-                    val = hp.values.index(trial_value)
-                else:
-                    val = trial_value
-                vector.append(val)
+                # Embed an HP value into the continuous space [0, 1].
+                prob = hp_module.value_to_cumulative_prob(trial_value, hp)
+                vector.append(prob)
+
+            if trial in ongoing_trials:
```

Comment:
The `ongoing_trials` dictionary is keyed by the tuner_id, not trial.
Are there unit tests for this method?